crates/bevy_render/src/camera/camera.rs

```diff
@@ -415,16 +451,16 @@ impl Camera {
     /// To get the world space coordinates with Normalized Device Coordinates, you should use
     /// [`ndc_to_world`](Self::ndc_to_world).
     ///
-    /// Returns `None` if any of these conditions occur:
-    /// - The logical viewport size cannot be computed. See [`logical_viewport_size`](Camera::logical_viewport_size)
-    /// - The near or far plane cannot be computed. This can happen if the `camera_transform`, the `world_position`, or the projection matrix defined by [`CameraProjection`] contain `NAN`.
-    ///     Panics if the projection matrix is null and `glam_assert` is enabled.
+    /// Panics if the camera's projection matrix is null and `glam_assert` is enabled
```

Comment:
I thought `NdcNan` is returned in that error case, a flag shouldn't change that, or am I getting something wrong?

Reply:
It's specifically the `inverse` operation in the `ndc` methods that would panic with that feature.

Reply:
Could we return the result in `world_to_nan` and have a check for the matrix being `is_nan` before the inversion instead of afterwards?